.readthedocs.yaml

```diff
@@ -13,7 +13,7 @@ build:
     python: "3.7"
   jobs:
     post_checkout:
-      - git remote set-branches origin master stackhpc/yoga stackhpc/xena stackhpc/wallaby
+      - git remote set-branches origin master stackhpc/2023.1 stackhpc/zed stackhpc/yoga stackhpc/xena stackhpc/wallaby
```

Comment:
Do the older branches need to know about later ones?

Reply:
I don't think so. Do you prefer the fix to be applied to each branch separately?

Reply:
I don't mind too much, but AFAICT each branch lists the earlier branches, so I was wondering if there is actually an issue to be fixed.

Reply:
I thought they didn't, I must have misread.